python/ray/util/iter.py

```diff
@@ -414,12 +414,17 @@ def base_iterator(timeout=None):
         name = "{}.batch_across_shards()".format(self)
         return LocalIterator(base_iterator, MetricsContext(), name=name)
 
-    def gather_async(self) -> "LocalIterator[T]":
+    def gather_async(self, async_queue_depth=1) -> "LocalIterator[T]":
         """Returns a local iterable for asynchronous iteration.
 
         New items will be fetched from the shards asynchronously as soon as
         the previous one is computed. Items arrive in non-deterministic order.
 
+        Arguments:
```

Comment:
Btw, is the google standard not "Args:" instead of "Arguments:"?

Reply:
That's true, will make a pr to bulk fix these later. We are currently using both everywhere.

Reply:
ok.